conf/application.conf

```diff
@@ -140,11 +140,6 @@ government-gateway {
   host = "http://localhost:9025"
 }
 
-feedback-survey-frontend {
```

Comment:
This config needs to be put back in as there are still values in frontendAppConfig relying on it (surveyUrl, signOutUrl).